### ISSUE TITLE
Actionsheet: Keyboard focus is not moving during navigation when JAWS is on and role of actionsheet is set to menu

### ISSUE THREAD

**Issue by pankajpandey01** [SAP/ui5-webcomponents-react#4075]
**Describe the bug**
Please see details here: https://github.com/SAP/ui5-webcomponents-react/issues/3887. 
This is reopening of above defect.

**Additional context**
Our a11y team expect ActionSheet should behave like 
https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/examples/menu-button-actions-active-descendant/

Stakeholder
Organization: {SuccessFactors}

**Comment by Lukas742** [comment]
Hi @pankajpandey01 

please add a detailed description of what problem you're facing and why the provided answer in the original issue is not sufficient.

**Comment by pankajpandey01** [comment]
As described in https://github.com/SAP/ui5-webcomponents-react/issues/3887, when JAWS screen reader is on, Keyboard focus is not moving during navigation. This issue happens when actionSheet role changed to menu. As per our a11y requirement, we need role as menu. Please provide solution with actionSheet role as menu.

**Comment by Lukas742** [comment]
I gave an explanation and solution to this problem in the original issue, and in my previous comment I kindly asked you to describe why that is not sufficient. So please do that.

**Comment by pankajpandey01** [comment]
Hi @Lukas742 
As per your solution, role="application" has to be enforced on application or component level.  We do not want role as application, we need role as menu for actionSheet as per a11y requirement. We can not set our page container role as "application" as page contain some non application normal web content.  Can you please suggest your solution by some example on top of https://codesandbox.io/s/react-playground-forked-zrm65p ?

**Comment by Lukas742** [comment]
Setting the role of the component to `application` doesn't necessarily mean that you cannot set another role to sub-elements. Therefore if you really only need this on component level, you can set it directly to the `ActionSheet`. 
If you're only consuming components from UI5 Web Components for React in your application, it's most probably better to globally attach this role, as ui5-webcomponents are accessible out of the box and don't need [traditional HTML interpretation techniques](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/application_role). You can do that for example directly on the `body` of the page, or on the root element of your React application.

[Here](https://codesandbox.io/s/react-playground-forked-ilfx66?file=/index.js:513-540) you can find an example where the role is only applied to the `ActionSheet` component.

**Comment by pankajpandey01** [comment]
Hi @Lukas742 
We want to update on component level. I tried above comment [example](https://codesandbox.io/s/react-playground-forked-ilfx66?file=/index.js:513-540) and still see original issue.
